cli/cmd/auth/login.go

```diff
@@ -64,7 +64,7 @@ func LoginCmd(cfg *config.Config) *cobra.Command {
 
 				fmt.Printf("Set default organization to %q. Change using `rill org switch`.\n", defaultOrg)
 			} else {
-				warn.Println("You are not part of any org. Run `rill org create` or `rill deploy` to create one.")
+				warn.Println("You are not part of any org. Run `rill org create` to create one.")
```

Comment:
Restore this – `rill deploy` also creates orgs

Reply:
OOPS got confused with multiple iterations.